flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/OverWindowITCase.scala

```diff
@@ -386,6 +386,28 @@ class OverWindowITCase extends BatchTestBase {
         row(5, 5)
       )
     )
+
+    // deal with input with 0 as the first row's rank field
+    checkResult(
+      "SELECT f, dense_rank() over (order by d) FROM Table5",
```

Comment:
If I don't misunderstood anything, your test seems to be wrong. 
The data of column d is `1, 2, 2, 3, 3, 3, 4, 4, 4, 4, 5, 5, 5, 5, 5`, how could it possible that the `dense_rank` on column d becomes `1, 2, 3, ..., 14, 15`?

Reply:
sorry, i made a mistake becase of careless

Reply:
it should be SELECT f, dense_rank() over (order by f) FROM Table5